org/pr/milestone.ts

```diff
@@ -15,19 +15,34 @@ declare var danger: DangerDSLType
 export declare function warn(message: string): void
 
 export async function checkMilestone() {
```

Comment:
This is a pure copy paste of the work from https://github.com/Automattic/peril-settings/pull/49/files

👇 👇 👇